tests/unittests/test_entry.py

```diff
@@ -43,6 +43,16 @@ def test_command_with_decode_utf_8():
     assert config.decode == ""
 
 
+def test_command_with_shell_pipeline():
+    from iredis.config import config
+
+    gather_args.main(["iredis", "--no-shell"], standalone_mode=False)
+    assert config.shell is False
```

Comment:
Can you also add a test, to check if the command line options actually overwrite the ~/.iredisrc config?

Just simply create a iredisrc and duplicated those two lines, and delete the rc file when test passed.

Reply:
That would also be good.

Reply:
Sorry I didn't make myself clear, I mean writing another test case, to verify that:

1. the command line options actually work
2. the command line options are higher priority

So let's write one more test function which exactlly identical to the original one.